easybuild/easyconfigs/r/ReFrame/ReFrame-3.7.3.eb

```diff
@@ -40,6 +40,9 @@ exts_list = [
 postinstallcmds = [
     "cp -r tutorials %(installdir)s",
     "mkdir -p %(installdir)s/share && cp -r share/completions %(installdir)s/share/completions",
+    # patch reframe script to disable import of 'site' module to make sure that Python packages installed
+    # in ReFrame installation directory are used, rather than those in the Python installation
+    r"sed -i 's@/\(python[0-9.]*\)$@/\1 -S@g' %(installdir)s/bin/reframe",
```

Comment:
I've checked and this change is causing the failure that I see. The Python being used is the one from my easybuild virtualenv.

`diff` on `bin/reframe` with and without this change.
```
< #!/rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/venv-dev-develop-up/bin/python -S
---
> #!/rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/venv-dev-develop-up/bin/python
```

Reply:
So adding the `-S` led to `zipfile` not being found anymore, because it's installed in your virtualenv?

Reply:
In the virtualenv
```
$ python -c "import zipfile; print(zipfile.__file__)"
/usr/lib64/python3.6/zipfile.py
```

Reply:
Would in this case help if you start python with `-S`, prepend `external` to the `sys.path` as is being done now and then call `site.main()`?